### PR TITLE
ticdc: update pulsar sink

### PR DESCRIPTION
**Pull request by overvenus**
### What is changed, added or deleted? (Required)

Update the Apache Pulsar sink configuration.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: N/A
- Other reference link(s): https://github.com/pingcap/ticdc/pull/751

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

**Comment by overvenus** [comment]
PTAL @TomShawn @amyangfei

File: ticdc/manage-ticdc.md [hunk 1]
```diff
@@ -111,64 +111,98 @@ Info: {"sink-uri":"mysql://root:123456@127.0.0.1:3306/","opts":{},"create-time":
 ```
 
 - `--changefeed-id`: 同步任务的 ID，格式需要符合正则表达式 `^[a-zA-Z0-9]+(\-[a-zA-Z0-9]+)*$`。如果不指定该 ID，TiCDC 会自动生成一个 UUID（version 4 格式）作为 ID。
-- `--sink-uri`: 同步任务下游的地址，需要按照以下格式进行配置，目前 scheme 支持 `mysql`/`tidb`/`kafka`。
+- `--sink-uri`: 同步任务下游的地址，需要按照以下格式进行配置，目前 scheme 支持 `mysql`/`tidb`/`kafka`/`pulsar`。
```
[the remark this answers is not in the document]
Comment (overvenus) [reply ticdc/manage-ticdc.md:114]: 我觉得没必要，因为开启预防在于 URI 中的参数，而不是 scheme。

[the remark this answers is not in the document]
Comment (amyangfei) [reply ticdc/manage-ticdc.md:114]: 那代码里 `+ssl` 的存在也没有意义了

**Review by amyangfei — COMMENTED** [review]
LGTM

**Review by leoppro — COMMENTED** [review]
LGTM

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**